Observer/NewShipment.php

```diff
@@ -143,6 +149,25 @@ private function setMagentoAndMyParcelTrack(Shipment $shipment): void
         $this->updateTrackGrid($shipment);
     }
 
+    /**
+     * @param $shipment
+     *
+     * @return void
+     * @throws \Exception
+     */
+    private function exportEntireOrder($shipment): void
+    {
+        $orderId = $shipment->getOrderId();
+
+        /**
+         * @var \Magento\Sales\Model\ResourceModel\Order\Collection $collection
+         */
+        $collection = $this->objectManager->get(MagentoOrderCollection::PATH_MODEL_ORDER);
+        $collection->addAttributeToFilter('entity_id', ['in' => $orderId]);
```

Comment:
Wanneer je toch naar deze pr kijkt, dubbelcheck / triplecheck dan even of deze $orderId inderdaad de entity_id is, en niet de increment_id